### ISSUE TITLE
Test coverage

### ISSUE THREAD

**Issue by sabeurthabti** [jasmine/jasmine-npm#40]
Has anyone managed to get testing coverage to work? I am using istanbul and I can't seem to get it to work. 

``` js
JASMINE_CONFIG_PATH='.utilities/jasmine' node_modules/.bin/istanbul cover --report json ./node_modules/.bin/jasmine
```

I don't get any istanbul output

**Comment by slackersoft** [comment]
What version of the jasmine npm are you using? There is an issue, #36, in 2.2.1. We are working on a cross-platform fix for the next release.

**Comment by sabeurthabti** [comment]
@slackersoft I am on Mac OSx, also 2.2.1.

**Comment by slackersoft** [comment]
Ok. For now I would use 2.2.0, since this issue should only be in 2.2.1. We're working on fixing this for good for the 2.3.0 release.

**Comment by sabeurthabti** [comment]
@slackersoft awesome man, 2.2.0 works great.

**Comment by amavisca** [comment]
Fixed with 97c67d5a0888b2694069dc56403e170203bad365
